exwm-manage.el

```diff
@@ -692,6 +706,8 @@ border-width: %d; sibling: #x%x; stack-mode: %d"
               #'exwm-manage--on-ConfigureRequest)
   (xcb:+event exwm--connection 'xcb:MapRequest #'exwm-manage--on-MapRequest)
   (xcb:+event exwm--connection 'xcb:UnmapNotify #'exwm-manage--on-UnmapNotify)
+  (when (eq system-type 'darwin)
```

Comment:
Please consider applying this 'workaround' to all platforms if it has no side-effect.  It would help reducing the testing scenarios.